src/components/RulesPage/rules.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable max-len */
+import * as Constants from "@niaefeup/nijobs-commons";
+
+export const rules = Object.freeze([
+    {
+        "title": "Using NIJobs (as a company)",
+        "description": "As a company, you must first apply for an account. The application will be reviewed and, if accepted, you will have an account registered in NIJobs. After that, you need to finish your company registration to be able to start posting job offers.",
+        "link": "/apply/company",
+        "linkName": "Apply as a company",
+    },
+    {
+        "title": "Creating an offer",
+        "description": `To create an offer you must first log in to your company account, then you can create an offer by pressing the "Create Offer" button. In order to avoid spam, and to ensure the greatest quality of offers to the students, a company cannot have more than ${Constants.Company.offers.max_concurrent} active offers at the same time. However, you can continue creating offers as long as they are hidden - you can set this in the advanced options when creating the offers. Your offers may be reviewed and removed in case they do not comply with the offer rules (see below). Creating duplicate entries will result in offer removal. If you believe you should have not received a penalty, please contact us with your problem and we will look into it. Offer creation can also be performed by a plaftorm administrator at a company's request.`,
```

Comment:
Don't talk about penalty, just state something like "If you believe your offer was disabled as a mistake, please contact.."
`disabled` instead of `removed`
duplicate offers should be moved to offer rules